src/pages/landing/LandingPage.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import landing_image_mellowhead from "../../assets/landing_page_images/landing_page_mellowhead.svg"
+import "./landingPage.css"
+import { Link } from "react-router-dom";
+
```

Comment:
If you separate the module import with local import will increase the readability
Ex:
//All module imports at the start
import React from "react";

//All local import
import "./landingPage.css"

Reply:
Got it. Thanks for the suggestion